Content.Shared/_RMC14/Xenonids/Parasite/VictimInfectedComponent.cs

```diff
@@ -71,4 +73,55 @@ public sealed partial class VictimInfectedComponent : Component
 
     [DataField, AutoNetworkedField]
     public EntityUid? Hive;
+
+    [DataField, AutoNetworkedField]
+    public int FinalStage = 5;
+
+    [DataField, AutoNetworkedField]
+    public int CurrentStage = 0;
+
+    [DataField, AutoNetworkedField]
+    public int InitialSymptomsStart = 2;
+
+    [DataField, AutoNetworkedField]
+    public int MiddlingSymptomsStart = 3;
+
+    [DataField, AutoNetworkedField]
+    public int FinalSymptomsStart = 4;
+
+    [DataField, AutoNetworkedField]
+    public float ShakesChance = 0.06f;
+
+    [DataField, AutoNetworkedField]
+    public float MinorPainChance = 0.02f;
+
+    [DataField, AutoNetworkedField]
+    public float ThroatPainChance = 0.02f;
+
+    [DataField, AutoNetworkedField]
+    public float MuscleAcheChance = 0.01f;
+
+    [DataField, AutoNetworkedField]
+    public float SneezeCoughChance = 0.02f;
+
+    [DataField, AutoNetworkedField]
+    public float MajorPainChance = 0.02f;
+
+    [DataField, AutoNetworkedField]
+    public TimeSpan BaseKnockdownTime = TimeSpan.FromSeconds(1);
+
+    [DataField, AutoNetworkedField]
+    public TimeSpan JitterTime = TimeSpan.FromSeconds(5);
+
+    [DataField, AutoNetworkedField]
+    public ProtoId<EmotePrototype> SneezeId = "Sneeze";
+
+    [DataField, AutoNetworkedField]
+    public ProtoId<EmotePrototype> CoughId = "Cough";
+
+    [DataField, AutoNetworkedField]
+    public ProtoId<EmotePrototype> ScreamId = "Scream";
+
+    [DataField, AutoNetworkedField]
+    public DamageSpecifier InfectionDamage = new() { DamageDict = new() { { "Blunt", 1 } } };
```

Comment:
These fields should be moved to a separate component or not be networked, otherwise when Dirty gets called or this component gets networked all of these fields will be sent to the client every time, making the component state very large.